flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/NettyMessage.java

```diff
@@ -84,7 +84,7 @@
 	 * 		{@link NettyMessage} subclass ID
 	 *
 	 * @return a newly allocated direct buffer with header data written for {@link
-	 * NettyMessageDecoder}
+	 * NettyMessageEncoder}
```

Comment:
This change is irrelevant and should be a separate hotfix commit.

Reply:
Have moved the fix into separate hotfix commit.